.asyncapi-tool

```diff
@@ -0,0 +1,14 @@
+title: AsyncAPI CLI
+description: |
+    One CLI to rule them all. 
+    This is a CLI that aims to integrate all AsyncAPI tools that you need while AsyncAPI document development and maintainance. 
+    You can use it to generate docs or code, validate AsyncAPI document and event create new documents.
+links:
+    websiteUrl: https://www.asyncapi.com/tools/cli
+filters:
+    language: CLI
+    technology:
+        - TypeScript
+    categories:
+        - others
```

Comment:
On the one hand, yes, but for CLI it's also important to have a info about language/technology, because without node we won't run our CLI - I don't know if oclif somehow bundles node with our cli and publishes as binaries anyway (I really don't know) - in C++ CLI you can publish binary and you don't need any external technology to run it. IMHO defining CLI as a language is wrong for me. I would prefer the language to actually be JavaScript, or TypeScript. In the technologies themselves, I would also specify JavaScript next to the TypeScript. WDYT? I don't want block you with other features, it's only mu suggestions.

Reply:
CLI do not require nodejs, there are already alternatives available, like brew and others (documented in readme)

`technology` is for specifying what technology was used to create it, in the end it was `TypeScript`. Do we really need to mention `JavaScript` is addition (I know one script is build with JS, but isn't it confusing really?)

Language is to specify for tools user to know what language is it created with, in case of integration. So for example you need to know that Generator language is `JavaScript` and technology is `JavaScript` and `React` and `Nunjucks`. So maybe in case of AsyncAPI CLI language should be empty?

makes sense?

Reply:
we will make it empty 😆

Reply:
@magicmatatjahu please approve

Reply:
I'm faster than you 😄